uo_sys_framework/StartInParachuteModule/cfgXEH/init/ClientPreInit.sqf

```diff
@@ -1,24 +1,24 @@
 #define COMPONENT StartInParachute
 #include "\x\UO_FW\addons\Main\script_macros.hpp"
 UO_FW_EXEC_CHECK(CLIENT);
-if (!UO_FW_Server_StartInParachuteModule_Allowed) exitWith {};
+
 ["UO_FW_SettingsLoaded", 
 {
-	
-	[{(CBA_missionTime > 1)}, 
+	if (!UO_FW_Server_StartInParachuteModule_Allowed) exitWith {};
```

Comment:
You can check server settings anywhere in the `UO_FW_SettingsLoaded` event handler. You don't need to add a waituntil and then check server version, you can do the check and exitwith before the waituntil is added